test/functional/rpc_getgeneralinfo.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the getgeneralinfo RPC."""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal, try_rpc
+
+class GetGeneralInfoTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 1
+
+    def run_test(self):
+        # Test getgeneralinfo
```

Comment:
Perhaps add these tests into an existing testfile like interface_bitcoin_cli.py.

Reply:
Would this mean that I should remove rpc_getgeneralinfo.py? Shouldn't every RPC call have it's own test file?

Reply:
Oh right, never mind, here it's implemented as an RPC and not the CLI. I conflated the two since it seemed like a CLI call :) About separate tests: We want the functional tests to run as quickly as possible, so if it's feasible to add this small test in a logical place in an existing setup, it might be better. That said, I wouldn't worry about it unless this PR gets Concept ACKs and others say the same.

Reply:
Thanks!
Ok, then I'll leave it as it is for now and will them come back when this PR gets enough ACKs.